index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="kr">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css" />
+    <link rel="stylesheet" href="/css/pages/home.css">
+    <title>판다마켓</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+            <img src="/assets/logos/logo_1.png" alt="판다마켓 로고">
+        </div>
+        <button class="login_btn" onclick="location.href='login.html'">로그인</button>
```

Comment:
💊 제안

해당 디자인이 버튼처럼 보이지만 기능상 페이지 이동이기에 a 태그 사용을 추천드립니다.
(디자인대로 태그를 작성하시기보다 역할을 보시고 태그를 선택하시는 것을 추천드려요.)
a 태그를 사용하시면 링크이기때문에 웹 접근성에도 좋으며 alt 를 누르고 링크 클릭시 새창으로 열 수도 있어 사용성면에서도 더 좋다고 생각합니당

```suggestion
        <a class="login_btn" href="/login.html">로그인</a>
```